content/docs/setup/terraform/_index.md

```diff
@@ -6,52 +6,274 @@ weight = 30
 toc = true
 +++
 
-### Automated setup using Terraform
+## Intro
 
 Using Terraform (with the Packet provider), create two servers _tf-provisioner_ and _tf-worker_ attached to the same VLAN.
 
-- Clone the _tink_ repository:
+The provisioner runs: `tink-server`, `boots`, `nginx to serve osie`, `hegel` and
+Postgres.
+
+First thins to do is to close the tink repository because it contains the
+Terraform file required to spin up the environment:
 
 ```
 git clone https://github.com/tinkerbell/tink.git
 cd tink/deploy/terraform
 ```
 
-- Update the `<packet_api_token>` and `<project_id>` fields in _input.tf_ with your Packet API token and desired project ID
-  - Make sure the API token is a user API token (created/accessed under _API keys_ in your personal settings)
-  - You may also update the hostnames in _main.tf_ if you prefer names other than _tf-provisioner_ and _tf-worker_
-- You may also update the hostnames in _main.tf_ if you prefer names other than _tf-provisioner_ and _tf-worker_
-- Run the following commands:
+This terraform module requires a couple of inputs, the mandatory one are the
+`packet_api_token` and the `project_id`. You can pass those to `terraform`
+command line tool via file, or inline with the flag `-var
+"project_id=235-23452-245-345"`.
+
+I have a file called `my_var.tfvars` that looks like this:
+
+```
+$ cat my_var.tfvars
+packet_api_token = "awegaga4gs4g"
+project_id = "235-23452-245-345"
+```
+
+Make sure the API token is a user API token (created/accessed under _API keys_ in your personal settings)
+
+Run the following commands:
 
 ```
-terraform init
-terraform apply
+$ terraform init -var-file=my_var.tfvars
```

Comment:
`terraform init --upgrade` should be used so the Packet provider is updated, if needed (the main.tf will define the version it should install).